### ISSUE TITLE
WCAG fixes on the summary component

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1029]
## Reading order
Today the reading order on the summary component is Label - "endre" button - Value. When using a screen reader this might be a little confusing. We should change it so that the button is the last element on summary element. 
**Example:** 
1. Is how it is today
2. How it should be
![Frame 1](https://user-images.githubusercontent.com/33222679/226929678-32e6e22f-310b-4af0-93f1-9db1bcbeb254.png)
The screen reader use the order of the code, if we change it we would have hack it to behave how we want it to. Maybe the easiest solution is to change the design so that the button is visually also the last element. 
![Skjermbilde 2023-03-22 kl  15 05 25](https://user-images.githubusercontent.com/33222679/226929817-e9b6b2ab-0d02-4289-8325-53cb9dc12d63.png)

- [ ] Change the design so that the "endre" button is last in the reading order

## The button is not descriptive enough
Moved to a separate issue:  #1100

### Steps To Reproduce

Go to an app with a summary page and use a screen reader when testing
